src/WorkerChannel.ts

```diff
@@ -61,4 +71,13 @@ export class WorkerChannel {
             after(context);
         }
     }
+
+    public async initAppDir(dir: string) {

```

Comment:
How about `updatePackageJson`

To me "init" means it only happens once

Reply:
Done